src/app/pages/news/11.3/04-notificationApi.md

```diff
@@ -1,6 +1,11 @@
 ## Business Notification API {#notificationAPI}
 
-... notifications
+Just introduced in 11.2, we now extended the Notifications feature even more.
+
+* Keep the users of a process up to date by using the new business notification API and event.
+* Notification messages have been streamlined, improved, and look better than ever.
+* All the information needed to fulfill your daily work is only a click away with the new link in notification messages to the details pages of the corresponding case or task. 
+* Customize the task notification message to the needs of your process by configuring the used notification template for a task.
```

Comment:
simpler to read?

> You can now customize the notification message of a task on its template.

Reply:
Trying to avoid using always "You can now ..."